qml/components/chatInformationPage/ChatInformationPageContent.qml

```diff
@@ -0,0 +1,459 @@
+    }
+    // header
+    PageHeader {
+        id: headerItem
+        z: 5
+        Item {
+            id: imageContainer
+            property bool hasImage: typeof chatInformationPage.chatInformation.photo !== "undefined"
+            property int minDimension: chatInformationPage.isLandscape ? Theme.itemSizeSmall : Theme.itemSizeMedium
+            property int maxDimension: Screen.width / 2
+            property int minX: Theme.horizontalPageMargin
+            property int maxX: (chatInformationPage.width - maxDimension)/2
+            property int minY: (parent.height - minDimension)/2
+            property int maxY: parent.height
+            property double tweenFactor: {
+                if(!hasImage) {
+                    return 0
+                }
+                return 1 - Math.max(0, Math.min(1, contentFlickable.contentY / maxDimension))
+            }
+
+            function getEased(min,max,factor) {
+                return min + (max-min)*factor
+            }
+            width: getEased(minDimension,maxDimension, tweenFactor)
+            height: width
+            x: getEased(minX,maxX, tweenFactor)
+            y: getEased(minY,maxY, tweenFactor)
+
+            ProfileThumbnail {
+                id: chatPictureThumbnail
+                photoData: imageContainer.hasImage ? chatInformationPage.chatInformation.photo.small : ""
+                replacementStringHint: headerItem.title
+                width: parent.width
+                height: width
+                radius: imageContainer.minDimension / 2
+                opacity: profilePictureLoader.status !== Loader.Ready || profilePictureLoader.item.opacity < 1 ? 1.0 : 0.0
+                optimizeImageSize: false
+            }
+            Loader {
+                id: profilePictureLoader
+                active: imageContainer.hasImage
+                asynchronous: true
+                anchors.fill: chatPictureThumbnail
+                source: chatInformationPage.isPrivateChat
+                        ? "../components/chatInformationPage/ChatInformationProfilePictureList.qml"
+                        : "../components/chatInformationPage/ChatInformationProfilePicture.qml"
+            }
+        }
+        // PageHeader changes the html base path:
+        property url emojiBase: "../js/emoji/"
+        leftMargin: imageContainer.minDimension + Theme.horizontalPageMargin + Theme.paddingMedium
+        title: chatInformationPage.chatInformation.title !== "" ? Emoji.emojify(chatInformationPage.chatInformation.title, Theme.fontSizeLarge, emojiBase) : qsTr("Unknown")
+        description: chatInformationPage.isPrivateChat ? ("@"+(chatInformationPage.privateChatUserInformation.username || chatInformationPage.chatPartnerGroupId)) : ""
+    }
+
+    SilicaFlickable {
+        id: contentFlickable
+        contentHeight: groupInfoItem.height + tabViewLoader.height
+        clip: true
+        interactive: true//groupInfoItem.height > pageContent.height * 0.5
```

Comment:
`interactive` should be false when scroll animation is running, e.g.
```qml
interactive: !scrollUpAnimation.running && !scrollDownAnimation.running
```

Reply:
Good catch!